### PR TITLE
MINOR: fix code listings for ops.html

### PR DESCRIPTION
**Pull request by jlprat**
Fixes indentation, uses the right character for bash (`>`), and fixes
syntax highlighting for listings

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by jlprat** [comment]
CC. @cadonna as you reviewed the last doc changes on formatting and styling, you might want to review these as well. I split the changes in several PRs so it's more bearable to review.

File: docs/ops.html [hunk 1]
```diff
@@ -248,98 +248,98 @@ <h5 class="anchor-heading"><a id="basic_ops_automigrate" class="anchor-link"></a
   For instance, the following example will move all partitions for topics foo1,foo2 to the new set of brokers 5,6. At the end of this move, all partitions for topics foo1 and foo2 will <i>only</i> exist on brokers 5,6.
   <p>
   Since the tool accepts the input list of topics as a json file, you first need to identify the topics you want to move and create the json file as follows:
-  <pre class="line-numbers"><code class="language-bash">  > cat topics-to-move.json
-  {"topics": [{"topic": "foo1"},
-              {"topic": "foo2"}],
-  "version":1
-  }</code></pre>
+  <pre class="line-numbers"><code class="language-bash">> cat topics-to-move.json
```
Comment (showuon) [inline docs/ops.html]: Should we replace `>` into `&gt;` here?

**Review by showuon — CHANGES_REQUESTED** [review]
@jlprat , thanks for the PR. Left a comment. Thanks.

File: docs/ops.html (hunk 1 above)
Comment (jlprat) [reply docs/ops.html]: Good catch! Pushed a fix

**Pull request closed without merging**